aidants_connect_web/views/espace_responsable.py

```diff
@@ -214,6 +217,41 @@ def remove_card_from_aidant(request, aidant_id):
     )
 
 
+@require_GET
+@login_required
+@user_is_responsable_structure
+@activity_required
+def remove_aidant_from_organisation(
+    request: HttpRequest, aidant_id: int, organisation_id: int
+) -> HttpResponse:
+    responsable: Aidant = request.user
+    aidant: Aidant = get_object_or_404(Aidant, pk=aidant_id)
+    organisation: Organisation = get_object_or_404(Organisation, pk=organisation_id)
+
+    if not responsable.can_see_aidant(aidant):
+        raise Http404
+
+    aidant_organisations = set(aidant.organisations.all())
+    if len(aidant_organisations) > 1 and organisation in aidant_organisations:
+
+        aidant.is_active = False
+        aidant.save()
+
+        django_messages.success(
+            request, f"Le profil de {aidant.get_full_name()} a été désactivé"
+        )
+    else:
+        aidant.organisations.remove(organisation_id)
```

Comment:
Ici ajouter quelque chose du type `if aidant.organisation not in aidant.organisations.all(): aidant.organisation = aidant.organisations.first()`